bootstrap.py

```diff
@@ -1053,12 +1082,71 @@ def exec_service(service, command, failonerror=True):
         migrate_systems(options.org, options.activationkey)
         if options.enablerepos:
             enable_repos()
+    elif options.new_capsule:
```

Comment:
if we end up here on a machine that is not registered at all yet, we get funny errors, should we check for a regged box?

Reply:
Is there a sane and simple way to do so?

Reply:
did you just ask for a sane way inside bootstrap.py? ;)

I've been playing around with

```python
    elif options.new_capsule:
        if not os.path.exists('/etc/rhsm/ca/katello-server-ca.pem'):
            print_generic("This system is not yet registered, cannot move to a new capsule")
            sys.exit(1)
```

for now.

probably even better:

```python
        if not os.path.exists('/etc/rhsm/ca/katello-server-ca.pem') or not os.path.exists('/etc/pki/consumer/cert.pem'):
```

Reply:
methinks that code should live in a function, too